src/main/java/trackitnus/logic/commands/lesson/ListLessonCommand.java

```diff
@@ -36,12 +37,15 @@ public ListLessonCommand(Optional<Code> codeOptional) {
     }
 
     @Override
-    public CommandResult execute(Model model) {
+    public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
         Predicate<Lesson> predicate;
         if (codeOptional.isEmpty()) {
             predicate = PREDICATE_SHOW_ALL_LESSONS;
         } else {
+            if (!model.hasModule(codeOptional.get())) {
+                throw new CommandException(MESSAGE_MODULE_DOES_NOT_EXIST);
+            }
```

Comment:
Actually I thought we were going to remove the ListLessonCommand, or are we leaving that to a later PR?

Reply:
Yep I will remove that in an upcoming PR, but for now just edit it 😂